website/projects/scimaps/src/app/shared/news-item/news-item.component.html

```diff
@@ -0,0 +1,8 @@
+<a class="news-item" [href]="item.pdfLink">
+  <img class="thumb" [src]="item.thumbnail" alt="image thumbnail">
+  <div class="info">
+    <div class="mat-display-1 date">{{item.date}}</div>
+    <div class="mat-subheading-1 title">{{item.title}}</div>
```

Comment:
No need for mat classes on title and identifier. They are automatically set to 14px! Only bold/light font-weight needs to be set in the theme file